defaults/main.yml

```diff
@@ -2,8 +2,17 @@
 # defaults file for ansible-coreos-kubelet
 coreos_kubelet_systemd: /etc/systemd/system
 coreos_kubelet_install_dir: /opt/bin
-kubelet_version: "v1.4.1_coreos.0"
+kubelet_version: "v1.7.0_coreos.0"
```

Comment:
The version here does not matter actually we can override it from the coreos-ansbile vars.